transformer.py

```diff
@@ -37,7 +37,10 @@ def create_vrt_permanent(base_dir: str, file_list: str, out_vrt: str = 'virtualT
         vrt_path = os.path.join(base_dir, out_vrt)
         cmd = 'gdalbuildvrt -srcnodata "-99 -99 -99" -overwrite -input_file_list ' + file_list +' ' + vrt_path
         logging.debug("Running Virtual Tiff command: '%s'", cmd)
-        os.system(cmd)
+        cmd_result = os.system(cmd)
+        # TODO: Should we raise an exception if the result is non-zero?
```

Comment:
Should these TODO's be in an issue? We'd need to see if the command return is consistent or not - sometimes it's just what's left on the stack that's returned; we'd need to test success & failure cases. Perhaps something for TravisCI?

Reply:
I'm not sure what the behavior _should_ be, I just noticed this and was wondering.

And I'm happy to remove these TODOs and turn them into issues if you prefer. I wanted to add them here so I wouldn't forget, but I don't like leaving things like this in the code.